Project-Bootcamp/components/table.js

```diff
@@ -0,0 +1,57 @@
+var React = require ('react');
+var _ = require ('lodash');
+
+var Table = React.createClass ({
+
+    propTypes: {
+        columns: React.PropTypes.array,
+        rows: React.PropTypes.array
+    },
+
+    getDefaultProps: function () {
+
+        return {
+            columns: [],
+            rows: []
+        };
+    },
+
+    render: function () {
+
+        return (
+            <table className="table table-striped">
```

Comment:
I wouldn't use *striped* as an identifier, since it's something that's related to a visual aspect of your UI and that could easily change. Maybe you have to think about the function that accomplishes more than its look.